core/src/main/kotlin/org/evomaster/core/problem/rest/OpenApiAccess.kt

```diff
@@ -62,17 +63,40 @@ object OpenApiAccess {
     }
 
     private fun readFromDisk(openApiUrl: String) : String {
+
+        // file scheme
         val fileScheme = "file:"
-        val path = if (openApiUrl.startsWith(fileScheme, true)) {
-            Paths.get(URI.create(openApiUrl))
-        } else {
-            Paths.get(openApiUrl)
+
+        // path variable
+        var path: Path? = null;
```

Comment:
why declaring it as nullable? was the compiler complaining?

Reply:
Hi Andrea,

Thank you so much for your comment. I changed the code in a way that it throws an exception if the path is null in the variable declaration as:

val path = {
} ?: throw SutProblemException("Could not set up the path: $openApiUrl")

So, if the path is null, it throws an exception mentioning that the path could not be set. I put null before because of the compiler complaining. The way I changed is suggested by IntelliJ.

Reply:
I have removed "?: throw SutProblemException("Could not set up the path: $openApiUrl")" and just used try-catch for exception handling.